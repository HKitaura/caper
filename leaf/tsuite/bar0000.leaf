require leaf;

fun main(): int
{
	puti(3*false+7);
	0;
}
