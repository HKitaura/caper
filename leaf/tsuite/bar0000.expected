leaf: context mismatch: <bool> appeared where <int> required; file: tsuite/bar0000.leaf, line: 5, column: 8
